### DISCUSSION TITLE
Composable undo package

### DISCUSSION THREAD

**Discussion by surajbarkale** [pointfreeco/swift-composable-architecture#782] [Show and tell]
I have published a new package https://github.com/aacapella/swift-composable-undo that implements undo/redo functionality. UndoManager is also supported but not required for the functionality.

**Comment by iampatbrown** [comment]
Awesome effort @surajbarkale. This looks pretty cool. I haven't used UndoManger much myself, but this seems like something people would appreciate for sure.

**Comment by stephencelis** [comment]
Just wanted to echo @iampatbrown's reaction. This definitely looks cool, @surajbarkale! Thanks for sharing :smile:

**Reply by surajbarkale** [reply]
I am using your work the template. So thanks for setting an example. I shall update the docs as I get some time.
